MicrosoftFluentUI.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'MicrosoftFluentUI'
-  s.version          = '0.26.0'
+  s.version          = '0.27.0'
```

Comment:
Version bumps + framework changes should be done in separate PRs

Reply:
updated